docker/Dockerfile

```diff
@@ -67,6 +65,7 @@ RUN pip3 install --upgrade pip
 RUN pip3 install -U mindmeld
 RUN pip3 install click-log==0.1.8
 
+ARG REBUILD_BLUEPRINT=unknown
```

Comment:
what is this?

Reply:
This is a trick to help rebuild docker container after a specific step

https://stackoverflow.com/questions/35154219/rebuild-docker-image-from-specific-step

Reply:
Otherwise, Docker will use a cached version of the assistant

Reply:
:)